### ISSUE TITLE
Is there any trick in training?

### ISSUE THREAD

**Issue by Alexuan** [Microsoft/human-pose-estimation.pytorch#48]
I tried to train the model by myself but it can not get the experiment results as you released no matter how I altered parameters.
Is there any trick in training? Or how to set the parameters?
Thank you!

**Comment by leoxiaobin** [comment]
There is no other trick for training. If you use 4 gpus and following our steps to train, you could exactly reproduce our results, no other modification needed.

**Comment by Alexuan** [comment]
I used 4 V100s and followed the steps to train, but the loss was fluctuating at 0.0012 and the mAP equaled to 0. Have you ever seen this phenomenon before?

**Comment by leoxiaobin** [comment]
Please follow the steps to disable cudnn of batchnorm.

**Comment by Scitator** [comment]
@leoxiaobin could you please explain why cudnn BN disabling is so important?

**Comment by leoxiaobin** [comment]
@Scitator ，for my understanding, I think that is a bug of cudnn. But we do not have the source code of cudnn, I can point which part cause the bug.

**Comment by Scitator** [comment]
Okay, but how have you get to the idea of cudnn BN disabling?
I mean, it quite unusual operation and if it does not work without it... it is just strange, you know. Have you tried to ask nvidia for help?

**Comment by leoxiaobin** [comment]
I found the mean variance of BN is not normal when I trained our net using cudnn, so I tried disabled BN of cudnn, then everything was normal.
I have not tried to ask NVIDIA for help now.
